hollow-ui-tools/src/main/java/com/netflix/hollow/ui/HollowDiffUtil.java

```diff
@@ -13,6 +12,7 @@ public static String formatBytes(long sizeInBytes) {
         sizeInBytes = Math.abs(sizeInBytes);
 
         int digitGroups = (int) (Math.log10(sizeInBytes)/Math.log10(1024));
+        DecimalFormat BYTE_FORMATTER = new DecimalFormat("#,##0.##");
```

Comment:
Ship it! 
nit: variable name casing.

Reply:
Good point, it used to be static final but it is renamed to just "formatter" now.